src/main/java/emu/grasscutter/data/GameDepot.java

```diff
@@ -18,6 +19,10 @@
 public class GameDepot {
     public static final int[] BLOCK_SIZE = new int[]{50,500};//Scales
 
+    public static final HashMap<Integer,ArrayList<BlossomReward>> BLOSSOM_REWARDS_GOLDEN = new HashMap<>();
+    public static final HashMap<Integer,ArrayList<BlossomReward>> BLOSSOM_REWARDS_BLUE = new HashMap<>();
+    public static final HashMap<Integer,ArrayList<Integer>> BLOSSOM_MONSTERS_DIFFICULTY = new HashMap<>();
```

Comment:
In Java, it's always best practice to use the base interface of a 'generic' object. (there's some fancy term for it but to sum it up) In this instance, the property type declaration would be `Map<Integer, List<BlossomReward>>`.